CMakeLists.txt

```diff
@@ -1,9 +1,7 @@
 cmake_minimum_required(VERSION 3.13)
```

Comment:
This PR seems to work fine. However, before merging we need to clean this up a little.